VirtoCommerce.Platform.Web/Converters/Diagnostics/InstalledModuleInfoConverter.cs

```diff
@@ -0,0 +1,26 @@
+using System.Linq;
+using Omu.ValueInjecter;
+using VirtoCommerce.Platform.Web.Model.Diagnostics;
+using VirtoCommerce.Platform.Web.Model.Modularity;
+using coreModel = VirtoCommerce.Platform.Core.Modularity;
+
+namespace VirtoCommerce.Platform.Web.Converters.Diagnostics
+{
+    public static class InstalledModuleInfoConverter
+    {
+        public static InstalledModuleInfo ToWebModel(this coreModel.ManifestModuleInfo moduleInfo)
+        {
+            var retVal = new InstalledModuleInfo();
+            retVal.InjectFrom(moduleInfo);
+            retVal.Version = moduleInfo.Version.ToString();
+            retVal.PlatformVersion = moduleInfo.PlatformVersion.ToString();
+            retVal.Groups = moduleInfo.Groups;
+            if (moduleInfo.Dependencies != null)
+            {
+                retVal.Dependencies = moduleInfo.Dependencies.Select(x => new ModuleIdentity { Id = x.Id, Version = x.Version.ToString() }).ToList();
```

Comment:
I guess we shouldn't use not readable variables in queries (like x, a, b.. etc). A much better use name of the variable in the singular, eg: .Dependencies.Select(**dependency** => new...)

Reply:
We have `x` as default for lambda expressions in our coding style now. For one variable - it is `x`. For more - we are using meaningful names. It is the current practice.
Though it is not reflected in our style guide, created the issue on it: https://github.com/VirtoCommerce/styleguide/issues/4

Reply:
Yes, you right in that the code will be shorter, but
It is not good practice for me, because:
1. Very often we have long query expressions and quite difficult to catch the real meaning of the variable if this var have name 'x' and etc.
2. If we have an iteration(like in the mentioned code) then each iteration item should have the name that is showing the singular meaning. 
3. If we doing the math calculation in the same code then we might get the name collisions.
4. The rules that you mentioned might be not obvious for common developers. It's +1 rule for memorizing.

Reply:
There could be pros and cons for any variant. We have this style accepted now. Maybe it could be changed after the team discussion, but I think we should follow our conventions before the change.

Reply:
Then we shouldn't add only Artem into reviewers for this, isn't? :)
https://github.com/VirtoCommerce/styleguide/issues/4

Reply:
Sure, you are free to add any VC members to the reviewers.
Code style discussion took place before, and with the issue, I'm asking Atrem to add this particular item in the way I remember it to the style guide.
To introduce another rule (or write it for the first time "to paper") I think it is better to raise the question and discuss it in e.g. our backend developers channel when you can ask whole VC dev team about this.

Reply:
Added PR about the current convention to have the starting point: https://github.com/VirtoCommerce/styleguide/pull/5.